index.html

```diff
@@ -8,28 +8,179 @@
+                    <li class="c-text"><h4> Back End Dev</h4></li>
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="languages">
+                    <li><a href="" class="btn">HTML</a></li>
+                    <li><a href="" class="btn">CSS</a></li>
+                    <li><a href="" class="btn">JavaScript</a></li>
+                </ul>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
+            </div>
+        </div>
+        <div class="card-container">
+            <header class="thumbnail">
+                <img src="src/img/Snapshoot Portfolio (2).png" alt="tonic2">
+            </header>
+            <div class="card">
+                <h2>Tonic</h2>
+                <ul class="info-card">
+                    <li class="canopy"><h3>CANOPY</h3></li>
+                    <li class="c-text"><h4> Back End Dev</h4></li>
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="languages">
+                    <li><a href="" class="btn">HTML</a></li>
+                    <li><a href="" class="btn">CSS</a></li>
+                    <li><a href="" class="btn">JavaScript</a></li>
+                </ul>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
+            </div>
         </div>
-        <div class="connect">
-            <p>LET'S CONNECT</p>
+        <div class="card-container">
+            <header class="thumbnail">
+                <img src="src/img/Snapshoot3.png" alt="multi-post2">
+            </header>
+            <div class="card">
+                <h2>Multi-Post Stories</h2>
+                <ul class="info-card">
+                    <li class="canopy"><h3>CANOPY</h3></li>
+                    <li class="c-text"><h4> Back End Dev</h4></li>
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="languages">
+                    <li><a href="" class="btn">HTML</a></li>
+                    <li><a href="" class="btn">CSS</a></li>
+                    <li><a href="" class="btn">JavaScript</a></li>
+                </ul>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
```

Comment:
[ OPTIONAL ] - [ ]

- [ ]  To ensure consistency with the Figma design, it is crucial to align the button interaction style accordingly. 🥱🥱

- [ ] It appears that your current button style does not match the Figma design. As per the project requirements, it is essential to strictly adhere to the specified button interaction style from the Figma design. 
- [ ] Please make the necessary adjustments to ensure that your button style strictly follows the Figma design, maintaining visual coherence and meeting the project requirements.
- [ ] Figma 
![Screenshot 2023-06-03 200911](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/118761303/7cf9fb4c-23a7-42df-ba6b-0deebbf5db62)